modules/haloIdSystem.js

```diff
@@ -42,7 +56,12 @@ export const haloIdSubmodule = {
    * @returns {IdResponse|undefined}
    */
   getId(config) {
-    const url = `https://id.halo.ad.gt/api/v1/pbhid`;
+    if (!utils.isPlainObject(config.params)) {
+      config.params = {};
+    }
+    const url = paramOrDefault(config.params.url,
+      `https://id.halo.ad.gt/api/v1/pbhid`,
+      config.params.urlArg);
```

Comment:
this parameter isn't documented in your .md file...should it be? what do you expect here?